content/sql/shop-project/_index.md

```diff
@@ -165,8 +166,16 @@ NB! Be sure to label you answers by putting the question above the code that ans
 
 - The following files should be present:
   - YAML/yml file with container setup.
-  - `src` directory with 4 SQL script files(a script to a database, create tables, populate tables and one to query the database).
-  - A new `.md` file needs to be present in the repo where the learner has documented what information is in which table and how the tables and which keys link the records between tables. They should not edit the `README.md` file because they didn't create it and it serves it's own purpose.
-
-    - topics/clean-code/sql
-- Did the learner adapt the naming conventions so that they matched those specified in {{< contentlink path="topics/clean-code/sql" >}}
\ No newline at end of file
+  - The `src` directory should contain 4 SQL script files: one to create the database, one to create tables, one to populate tables, and one to query the database.
+  - A new .md file needs to be present in the repository where the learner has documented what information is in each table, how the tables are related, and which keys link the records between tables. The README.md file should not be edited, as it serves its own purpose.
+  
+  **NB! Many learners configure the docker-compose.yaml file to make use of the Adminer image but still instruct reviewers to download and install Postgres on their local machines in the documentation. This indicates a misunderstanding of the relationship between these technologies.**
+
+  To Clarify:
+  - Docker allows you to run applications in isolated containers.
+  - Docker Compose is a tool for defining and running multi-container Docker applications.
+  - Using Docker and Docker Compose, you can set up your entire environment, including Postgres and Adminer, without requiring any local installations on the reviewer’s machine.
+  
+  Ensure the documentation reflects this setup correctly, so reviewers do not need to install Postgres locally. Refer to the following resource for more information: [How to Set Up Postgres and Adminer with Docker Compose](https://codepruner.com/how-to-run-postgresql-and-adminer-or-pgadmin-with-docker-compose/)
```

Comment:
This part seems is important and should probably be higher up in the content, maybe under the `Installation` information.